### ISSUE TITLE
Add Geolocation Of Libyan Cities

### ISSUE THREAD

**Issue by omarhagoche** [omarhagoche/LibyaCities#1]
إضافة احداثيات المدن الليبية إلى ملف json الخاص بمدن ليبيا

/libya/cities.json
https://github.com/omarhagoche/omarhagoche.github.io/blob/main/libya/cities.json

**Comment by arousi** [comment]
Fixed in PR #7

**Comment by omarhagoche** [comment]
> Fixed in PR [#7](https://github.com/omarhagoche/LibyaCities/pull/7)

No, it's not, the PR, is different and not an hotfix on this issue
PR #7 is incomplete work plan
